profile/src/main/java/rdx/works/profile/domain/ProfileException.kt

```diff
@@ -2,11 +2,11 @@ package rdx.works.profile.domain
 
 import rdx.works.profile.data.model.pernetwork.Entity
 
-sealed class ProfileException : Exception() {
+sealed class ProfileException(msg: String? = null, cause: Throwable? = null) : Throwable(msg, cause) {
```

Comment:
The problem with ProfileException is that it cannot abide by the RadixWalletException since we have no "domain" extracted into its own module. What if we moved `RadixWalletException` into `core`?

Reply:
If I would have this `ConnectorExtensionThrowable` interface implemented by some of subtypes of RadixWalletException, we would also need dapp/model/WalletErrorType accessible in core, which does not seem ideal. Without implementing that interface and keeping this `toWalletErrorType` extension, I could move it to core and make ProfileException subtype of Radix one.

Reply:
Ahh, it is a pity that we have some parts of the code modularised :(